core/src/main/java/me/mykindos/betterpvp/core/combat/click/RightClickListener.java

```diff
@@ -37,13 +38,28 @@ public class RightClickListener implements Listener {
     private final ClientManager clientManager;
     private final Core core;
     private final Map<Player, RightClickContext> rightClickCache = new HashMap<>();
+    private final WeakHashMap<Player, Long> lastDrop = new WeakHashMap<>();
 
     @Inject
     public RightClickListener(ClientManager clientManager, Core core) {
         this.clientManager = clientManager;
         this.core = core;
     }
 
+    // Fix for interact event triggering when dropping items
+    @EventHandler
+    public void onDrop(PlayerDropItemEvent event) {
+        lastDrop.put(event.getPlayer(), System.currentTimeMillis());
+    }
```

Comment:
Is this necessary still? Or is this part of the eventresult.deny stuff

Reply:
Yeah it's part of that bugfix. I think an alternative is to mess with server internals but that's probably a lot more work than we think